yt_dlp/extractor/acfun.py

```diff
@@ -139,28 +139,28 @@ def _real_extract(self, url):
         json_bangumi_data = self._search_json(r'window.bangumiData\s*=\s*', webpage, 'bangumiData', video_id)
 
         if not mobj.group('ac_idx'):
-            video_info = json_bangumi_data['currentVideoInfo']
-            playjson = self._parse_json(video_info['ksPlayJson'], video_id)
-            title = json_bangumi_data['showTitle']
+            json_key, title_key = 'currentVideoInfo', 'showTitle'
         else:
             # if has ac_idx, this url is a proxy to other video which is at https://www.acfun.cn/v/ac
             # the normal video_id is not in json
             ac_idx = mobj.group('ac_idx')
             video_id = f"{video_id}_ac={ac_idx}"
-            video_info = json_bangumi_data['hlVideoInfo']
-            playjson = self._parse_json(video_info['ksPlayJson'], video_id)
-            title = video_info['title']
+            json_key, title_key = 'hlVideoInfo', 'title'
+
+        video_info = json_bangumi_data[json_key]
+        playlist_json = self._parse_json(video_info['ksPlayJson'], video_id)
```

Comment:
My bad, is this refer to `playlist` ? if not, you should change back the name to `playjson` to avoid confusion

Reply:
yes, it is playlist in the last version